extensions.json

```
@@ -1130,6 +1135,11 @@
       "id": "twxs.cmake",
       "repository": "https://github.com/twxs/vs.language.cmake"
     },
+    {
+      "id": "Unity.unity-debug",
+      "repository": "https://github.com/Unity-Technologies/vscode-unity-debug",
+      "version": "3.0.0"
```

```suggestion
      "version": "3.0.0",
      "checkout": "Version-3.0.1"
```
The mismatch is a bit strange here, though.